Dockerfile

```diff
@@ -124,9 +124,11 @@ RUN apt-get update && apt-get install -y unzip wget
 WORKDIR /workspace/machines
 # Download old WASM module roots
 RUN bash -c 'mkdir 0x21f708e444c3afb7689fa5d0737b3942fd19012c0081d359ba3d59b7643d7810 && cd $_ && wget https://github.com/OffchainLabs/nitro/releases/download/devnet-consensus-v1/machine.wavm.br'
-RUN bash -c 'mkdir 0xb7905959ec167e0777bbbd6c339b0c98d676729cb502722aa01a34964f817ca3 && cd $_ && wget https://github.com/OffchainLabs/nitro/releases/download/devnet-consensus-v2/machine.wavm.br'
+RUN bash -c 'mkdir 0xb7905959ec167e0777bbbd6c339b0c98d676729cb502722aa01a34964f817ca3 && LATEST=$_ && ln -s $LATEST latest && cd $LATEST && wget https://github.com/OffchainLabs/nitro/releases/download/devnet-consensus-v2/machine.wavm.br'
 # Copy in latest WASM module root
-COPY --from=module-root-calc /workspace/target/machines/latest latest
+WORKDIR /workspace/latest-machine
```

Comment:
don't have latest-machine in this docker. It creates an unnecessary dependency that will force ocker to build the entire WASM machine for nitro-node.
When building the non-dev nitro-node, we don't need to build any WASM.

Reply:
:+1: